lib/common.sh

```diff
@@ -293,6 +293,8 @@ else
   export CAPM3_IMAGE="${CAPM3_IMAGE:-${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:main}"
   export IPAM_IMAGE="${IPAM_IMAGE:-${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:main}"
   export BMOBRANCH="${BMORELEASEBRANCH:-main}"
+  export KEEPALIVED_TAG="v0.6.1"
```

Comment:
These are added for testing, or actually part of the change?

Reply:
These were for debugging. I removed them.